packages/react-aria/src/hooks/useARIAButton.stories.tsx

```diff
@@ -1,11 +1,11 @@
 import { getSlots } from '@fluentui/react-utilities';
 import * as React from 'react';
 import { useARIAButton } from './useARIAButton';
-import type { ComponentState, IntrinsicSlotProps } from '@fluentui/react-utilities';
+import type { ComponentState, Slot } from '@fluentui/react-utilities';
 import type { ARIAButtonSlotProps } from './useARIAButton';
 
 type Slots = {
-  root: IntrinsicSlotProps<'div'>;
+  root: Slot<'div'>;
   button: ARIAButtonSlotProps;
```

Comment:
Should be `ARIAButtonSlotProps` renamed to something else?

Reply:
That type name is ok IMO, but good catch here: this should be `Slot<ARIAButtonSlotProps>`. I'll update this and other places that do the same thing.